app/javascript/components/profile/recommendations.vue

```diff
@@ -1,15 +1,20 @@
 <template>
-  <div class="profile-recommendations">
-    <profile-recommendations-users
-      :being-fetched="fetchingRecommendations"
-      :recommendations="recommendations"
-      type="best"
-      :title="$i18n.t('message.profile.recommendedReviewers')" />
-    <profile-recommendations-users
-      :being-fetched="fetchingRecommendations"
-      :recommendations="recommendations"
-      type="worst"
-      :title="$i18n.t('message.profile.notRecommendedReviewers')" />
+  <div class="profile-interactions">
+    <profile-relations
+      :beingFetched="fetchingRecommendations"
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Attribute ':beingFetched' must be hyphenated                            vue/attribute-hyphenation